Tools/clinic/clinic.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
-#
```

Comment:
I don't think Clinic needs a locally built Python.

Reply:
Yes. It's don't need a local builtin python. There have docs to introduce how to run this clinic.py. So I think remove it would be better(Avoid update the python version again an again).